kyo-core/shared/src/test/scala/kyo/LogTest.scala

```diff
@@ -22,21 +22,6 @@ class LogTest extends Test:
         end for
     }
 
-    "unsafe" in {
-        import AllowUnsafe.embrace.danger
-        Log.live.unsafe.trace("trace")
-        Log.live.unsafe.debug("debug")
-        Log.live.unsafe.info("info")
-        Log.live.unsafe.warn("warn")
-        Log.live.unsafe.error("error")
-        Log.live.unsafe.trace("trace", ex)
-        Log.live.unsafe.debug("debug", ex)
-        Log.live.unsafe.info("info", ex)
-        Log.live.unsafe.warn("warn", ex)
-        Log.live.unsafe.error("error", ex)
-        succeed
-    }
-
```

Comment:
You can keep this if you keep live. Otherwise you should add an equivalent using the unsafe API